app/src/components/widgets/singleLevelWidgets/charts/common/passingRateChart/config/getConfig.js

```diff
@@ -73,6 +90,8 @@ export const getConfig = ({
       contents: createTooltipRenderer(IssueTypeStatTooltip, calculateTooltipParams, {
         totalItems,
         formatMessage,
+        includeSkipped,
+        totalItemsWithoutSkipped,
```

Comment:
Can we provide just `totalItems` here and don't change tooltip calculation?
f.e.
```javascript
totalItems: includeSkipped ? totalItems : totalItemsWithoutSkipped ,
```